content/blogs/YYYY-MM-DD-CI-CD-kestra-comparison.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Kestra vs. Popular CI/CD Tools: When to Choose an Orchestration Solution"
+description: "Learn when to to choose an orchestrator rather than a CI/CD solution"
+date: xxxxxxx
+category: Solutions
+author:
+  name: Federico Trotta
+  image: "ftrotta"
+image: /blogs/xxxxx.jpg
+---
+
+In a recent [blog](./2024-09-18-what-is-an-orchestrator.md), we've defined what an orchestrator is, what are the differences between orchestration and automation, and how orchestration can help you to automate your workflows.
+
+Now, it's time to answer the question: when should you choose an orchestrator over a CI/CD solution?
+
+In this article, we'll compare Kestra to popular CI/CD solutions to help you get a practical understanding of when to choose an orchestrator over a CI/CD tool. In particular, at the end of the article, we'll provide a deep comparison between Kestra and Jenkins.
+
+Ready? Let's dive in!
+
+## Comparing Kestra to CI/CD Tools
+As we've already clarified the difference between orchestration and automation in the above-mentioned blog, let’s directly dive into how Kestra stacks up against popular CI/CD tools. 
+
+### Kestra Vs. GitHub Actions
+[GitHub Actions](https://github.com/features/actions) is the go-to for many developers because it’s embedded right into GitHub, making it super convenient for CI/CD as it allows to build, test, and deploy code right from GitHub: basically, anything developers needs to do in their day-to-day work.
+
+It is also developer-friendly, easy to set up, and integrates well with your codebase. 
+
+![A use of GitHub Actions by Federico Trotta](/blogs/YYYY-MM-DD-CI-CD-kestra-comparison/github_actions.png)
+
+But where it excels in simplicity, it lacks in handling workflows that need to span across multiple systems, such as deploying applications on various platforms like AWS, Azure, or Google Cloud Platform (GCP), and this is where Kestra shines.
+
+In fact, if your workflow goes beyond just automating builds and deployments  — let’s say you need to automate data processing tasks across cloud services — here's where Kestra’s orchestration capabilities really shine. Kestra, in fact, can handle more complex workflows with dependencies between services, and it also provides mechanisms for error handling and managing events - even [in real time](./2024-06-27-realtime-triggers.md).

```

Comment:
This is really vague so hard for the reader to understand. Can you give a proper example where this might not work on GitHub Actions but does work on Kestra. Or maybe works on both, but is easier with Kestra.

Note: we don't need to just point out things the alternatives don't do, focus on what we do better / easier.